pkg/test/framework/tools/featuresgen/cmd/root.go

```diff
@@ -1,219 +0,0 @@
-// Copyright Istio Authors
```

Comment:
This is all deadcode but used the legacy libraries; removed entirely instead of fixing it